### ISSUE TITLE
the files in channel is not accessible during the keyboard navigation

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#2112]
@CoBC when i go to the files tab and focus on the files list and then look backward with object navigation, i see this text, wich should be accessible using the keyboard as whell, by pressing shift + tab once when you are in files list, so between files list and files tab: this is the text.
Files in channel: /audio box/
this is useful to show the channel path for example when you are an admin and you wish to modify the files of other channel that you are not joined, and for default users to see the channel path to make sure what channels they are modifying.
